CMSgov/eAPD
# [Sev1] NumberField Bug

Resolves #4161 

### Description
Allows up to 1/100th of a percent to be input in FTE fields by allowing inputs `0.0` and `0.00`. Beyond that the previous behavior will occur where `0.000` will immediately convert to `0`

### Signifcant changes or possible side effects
* This is a change to NumberField so it's possible there is an impact to another part of the app.

### Automated test cases written
* Added tests to handle the new allowed cases.

### Steps to manually verify this change

1. Navigate to an FTE field
2. Enter a decimal with a leading zero such as .01
3. Verify it enters successfully

### This pull request is ready to review when


### This pull request can be merged when
